### PR TITLE
Create BEST_practices_wg_graduation_stage.md

### PR DESCRIPTION
**Pull request by SecurityCRob**
PR to recognize BEST WG is "graduated" in lifecycle

### CODE REVIEW

**Review by sevansdell — APPROVED** [review]

**Review by steiza — APPROVED** [review]

**Review by lehors — CHANGES_REQUESTED** [review]
LGTM but as indicated in our instructions, this PR should also change the status of the TI in the README, otherwise we'll had to do it in another PR.
https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#working-group-creation-or-change-of-lifecycle-stage

File: process/wg-lifecycle-documents/BEST_practices_wg_graduation_stage.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- [Daniel Applequist*, Snyk](https://github.com/Torgo)
+- [Georg Kunz, Ericsson](https://github.com/gkunz)
+- [Sarah Evans, Dell](https://github.com/sevansdell) - Security Toolbelt SIG Co-Lead
+- [John Kjell, TestifySec](https://github.com/jkjell) - Security toolbelt SIG Co-lead
+- [Thomas Nyman, Ericsson](https://github.com/thomasnyman) - C/C++ Compiler Guide SIG Lead
+- [Nell Shamrell-Harrington](https://github.com/nellshamrell) - Memory Safety SIG Lead
+- [Avishay Balter, Microsoft](https://github.com/balteravishay) - Memory Safety SIG Co-Lead
+
+
+### Governance
```
Comment (marcelamelara) [inline process/wg-lifecycle-documents/BEST_practices_wg_graduation_stage.md:24]: Can you please add a link in here that points to the project governance documents (per the https://github.com/ossf/tac/pull/296 TIs Gives+Gets and updated WG template)?

**Review by marcelamelara — CHANGES_REQUESTED** [review]
To align with the other TI graduate requirements for governance docs, can you please add a link in this application?

**Comment by lehors** [comment]
> LGTM but as indicated in our instructions, this PR should also change the status of the TI in the README, otherwise we'll had to do it in another PR. https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#working-group-creation-or-change-of-lifecycle-stage

I fixed it for you. :-)

**Review by lehors — APPROVED** [review]
LGTM

File: process/wg-lifecycle-documents/BEST_practices_wg_graduation_stage.md (hunk 1 above)
Comment (SecurityCRob) [reply process/wg-lifecycle-documents/BEST_practices_wg_graduation_stage.md:24]: Added link to WG charter - https://github.com/ossf/wg-best-practices-os-developers/blob/main/CHARTER.md

**Review by marcelamelara — APPROVED** [review]
Thanks CRob !

**Comment by SecurityCRob** [comment]
5 of 9 TAC members have approved this PR, merging.  Congrats on reaching "Graduated" level, BEST WG!

**Pull request merged**